Policies/github-terms-of-service.md

```diff
@@ -273,7 +276,7 @@ It is your responsibility to properly cancel your account with GitHub. You can [
 #### 2. Upon Cancellation
 We will retain and use your information as necessary to comply with our legal obligations, resolve disputes, and enforce our agreements, but barring legal requirements, we will delete your full profile and the Content of your repositories within 90 days of cancellation or termination (though some information may remain in encrypted backups). This information can not be recovered once your account is cancelled.
 
-We will not delete Content that you have contributed to other Users' repositories or that other Users have forked.
+We will not delete Content that you have contributed to Other Users' repositories or that Other Users have forked.
```

Comment:
> Unrelated, should be split into a separate PR.

Split off into #57.  I'll rebase this one (either to drop “Other Users” or to put this commit on top of #57) once there's enough feedback to guess whether #57 will be rejected or accepted.